hello-vulkan/app/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+apply plugin: 'org.jetbrains.kotlin.android'
+
+
+android {
+    compileSdkVersion 33
+    ndkVersion '25.1.8937393'
+    
+    defaultConfig {
+        shaders {
+            glslcArgs.addAll(['-c'])
+        }
+        applicationId 'com.android.hellovk'
+        minSdkVersion 30
+        targetSdkVersion 33
+        externalNativeBuild {
+            cmake {
+                // Available arguments are inside ${SDK}/cmake/.../android.toolchain.cmake file
+                arguments "-DANDROID_STL=c++_static"
+            }
+        }
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled = false
```

Comment:
probably leave this as true? then app minimizing will happen for release build and could find potential errors.  hello-jni probably should also be changed back to "true" too.  I can help for that. 

if proguard-rules.pro is empty, no need to use it either in the next statement.